utils/validations.js

```
@@ -95,6 +95,10 @@ exports.isWebMaster = (currentUser) => {
  */
 exports.isValidTrackingScript = (script) => {
   const dummyString = process.env.DUMMY_STRING;
+  if(!dummyString){
+    dummyString = 'Error on script validation'
```

Biome (1.9.4)

[error] Can't assign dummyString because it's a constant

This is where the variable is defined as constant

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)